feature/account/setup/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="account_setup_title">K-9 Mail</string>
+    <string name="account_setup_top_bar_subtitle">K-9 Mail</string>
```

Comment:
Things to remember for later: App-specific strings should live in a separate file.

Reply:
Right, so far they live in legacy and are not accessible here, maybe lift some of the strings into a core resource module.